src/overlays/actors/ovl_En_Snowwd/z_en_snowwd.c

```diff
@@ -30,25 +30,107 @@ ActorInit En_Snowwd_InitVars = {
     (ActorFunc)EnSnowwd_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80AF7A90 = {
-    { COLTYPE_TREE, AT_NONE, AC_ON | AC_HARD | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK5, { 0x00000000, 0x00, 0x00 }, { 0x0100020A, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_TREE,
+        AT_NONE,
+        AC_ON | AC_HARD | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK5,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x0100020A, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 18, 60, 0, { 0, 0, 0 } },
 };
 
-#endif
+Vec3f D_80AF7ABC = { 0.0f, 0.0f, 0.0f };
 
-extern ColliderCylinderInit D_80AF7A90;
+Vec3f D_80AF7AC8 = { 0.0f, -4.0f, 0.0f };
 
-extern UNK_TYPE D_06001AA0;
+Color_RGBA8 D_80AF7AD4 = { 255, 255, 255, 255 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/EnSnowwd_Init.s")
+Color_RGBA8 D_80AF7AD8 = { 200, 200, 220, 0 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/EnSnowwd_Destroy.s")
+void EnSnowwd_Init(Actor* thisx, PlayState* play) {
+    EnSnowwd* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/func_80AF76F0.s")
+    this->actor.home.rot.z = 0;
+    this->actor.home.rot.y = 0;
+    this->unk190 = 0;
+    this->actor.uncullZoneForward = 4000.0f;
+    this->actor.uncullZoneScale = 2000.0f;
+    this->actor.uncullZoneDownward = 2400.0f;
+    Collider_InitAndSetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = func_80AF76F0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/EnSnowwd_Update.s")
+void EnSnowwd_Destroy(Actor* thisx, PlayState* play) {
+    EnSnowwd* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Snowwd/EnSnowwd_Draw.s")
+    Collider_DestroyCylinder(play, &this->collider);
+}
+
+void func_80AF76F0(EnSnowwd* this, PlayState* play) {
+    s32 pad;
+    Actor* thisx = &this->actor;
+    f32 sp54;
+    Vec3f sp48;
+
+    if (this->collider.base.acFlags & AC_HIT) {
+        this->collider.base.acFlags &= ~AC_HIT;
+        Actor_PlaySfx(thisx, NA_SE_IT_REFLECTION_WOOD);
+    }
+    if (thisx->home.rot.y != 0) {
+        this->unk190 = 0x15;
+        thisx->home.rot.y = 0;
+        if (thisx->home.rot.z == 0) {
+            if (SNOWWD_GET_PARAM_F80(&this->actor) < 0x10) {
+                sp48 = thisx->world.pos;
+                sp48.y += 200.0f;
+                Item_DropCollectibleRandom(play, NULL, &sp48, (SNOWWD_GET_PARAM_F80(&this->actor) * 0x10));
```

Comment:
```suggestion
                Item_DropCollectibleRandom(play, NULL, &sp48, SNOWWD_GET_PARAM_F80(&this->actor) * 0x10);
```
Extra bracket not needed